frontend/src/components/DonatorHomePage/DonatorHomePage.css

```
@@ -0,0 +1,107 @@
+#donatePage {
+    text-align: center;
+}
+
+#giftBox {
+    height: 350px;
+    width: 500px;
+    margin: auto;
+    margin-top: 40vh;
+    transform: translateY(-50%);
+}
+
+#donatePageGift {
+    width: 150px;
+}
+
+#donatePageHeader {
+    color: #314D89;
+}
+
+#donatePageDesc {
+    font-size: 15px;
```

```suggestion
    font-size: 15px;
    padding-bottom: 10px;
```
Adding a little spacing between the button and text.